src/main/java/goodgenerator/blocks/tileEntity/PreciseAssembler.java

```diff
@@ -95,7 +93,7 @@ public IStructureDefinition<PreciseAssembler> getStructure_EM() {
                                             OutputBus,
                                             Maintenance,
                                             Muffler,
-                                            HatchElement.EnergyMulti.or(Energy))
+                                            ExoticEnergy)
```

Comment:
Why not Energy.or(ExoticEnergy)? From my understanding this will prevent normal hatches from working or was there a change?

Reply:
normal hatch still can work but it won't support survival auto bulid with normal hatch